src/containers/Home.js

```diff
@@ -295,39 +255,61 @@ class Homepage extends React.PureComponent {
             { property: 'og:url', content: siteMeta.urlOrigin + '/' },
           ]}
         />
-        <SideBar ref={this._sidebar} anchors={anchors}>
-          <LatestSection data={this.props[fieldNames.sections.latestSection]} />
-          <EditorPicks
-            data={this.props[fieldNames.sections.editorPicksSection]}
-          />
-          {latestTopicJSX}
-          <DonationBoxSection />
-          <ReviewsSection
-            data={this.props[fieldNames.sections.reviewsSection]}
-            moreURI={`categories/${categoryConst.pathSegments.reviews}`}
-          />
-          <NewsLetterSection />
-          <Background backgroundColor={moduleBackgounds.category}>
-            <CategorySection data={this.props.categories} />
-          </Background>
-          <Background backgroundColor={moduleBackgounds.topic}>
-            <TopicsSection
-              data={this.props[fieldNames.sections.topicsSection]}
+        <SideBar ref={this._sidebar}>
+          <Section anchorId="latest" anchorLabel="" showAnchor={false}>
```

Comment:
minor. see here for [reference](https://github.com/twreporter/twreporter-npm-packages/pull/186#discussion_r630725234)

Reply:
Updated.